popSimManu.tex

```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
-(Figure~\ref{fig:n_t_sheehan}). In the two-population setting for human
+(Figures~\ref{fig:n_t_ragsdale} and~\ref{fig:n_t_gutenkunst}), which
+tends to over-estimate the size of oscillations.  By contrast,
+\stairwayplot outperforms the sequence-based methods
+on simulations of \textit{D.~melanogaster} populations,
+in which linkage disequilibrium is considerably reduced (Figure~\ref{fig:n_t_sheehan}).
+In simulations of two human populations
 (Figure~\ref{fig:IM_popn_human}), most methods do reasonably well at
 reconstructing the YRI history, but struggle with the more complex CEU
 history, in large part because of the restriction of constant sizes per
 population.  An exception is \smcpp, which does not have the same
 restrictions on its inferred history, and as a result does somewhat better
 with the CEU history but tends to overfit the YRI history.  The results for
-the two-population {\em Drosophila} case (Figure~\ref{fig:two_popn_fly})
+the two-population \textit{D.~melanogaster} case (Figure~\ref{fig:two_popn_fly})
 are generally similar.  
 
 Altogether, these preliminary experiments highlight
 the utility of \stdpopsim for comparing a variety of inference methods on
 the same footing, under a variety of different demographic models.
 In addition, the ability of \stdpopsim to generate data with and without significant features, such
-as the genetic map or population size change (e.g., Figure \ref{fig:n_t_HomSap_constant}), allows
+as a genetic map or population size change (e.g., Figure \ref{fig:n_t_HomSap_constant}), allows
 investigation of the failure modes of popular methods.
 Moreover the comparison of methods across the various genome organizations, genetic maps,
 and demographic histories of different organisms, provides valuable information
 about how methods might perform on non-human systems.
 
 Finally, comparison of results across methods or simulation runs
 provides an estimate of inference uncertainty, analogous to parametric
-bootstrapping.
+bootstrapping,
+especially since different methods are likely vulnerable to model misspecification
+in different ways.
 
 \stdpopsim is intended to be a fully open, community-developed project.  
 Our implementations of genome representations and genetic maps for the some of
-the most common study systems in computational genetics---humans, \emph{Drosophila},
-and \emph{Arabidopsis} (among others)---are only intended to be a starting point for
-future development.  Indeed we envision future work to incorporate common modeling
-violations such as selection, gene conversion, and mutational heterogeneity, 
+the most common study systems in computational genetics---humans, \textit{Drosophila},
+and \textit{Arabidopsis} (among others)---are only intended to be a starting point for
+future development.  In addition to other taxa,
+we hope to soon incorporate other common biological processes
+such as selection, gene conversion, and mutational heterogeneity, 
```

Comment:
doesn't quite work with the rest of this sentence. rephrase?

Reply:
oh whoops

Reply:
fixed